### DISCUSSION TITLE
How can participatory mapping and critical cartography be improved?

### DISCUSSION THREAD

**Discussion by ninalong** [albertkun/21S-ASIAAM-191A#77] [Past Thinking Caps]
In "Participatory Mapping," Chambers lists a number of ethical concerns for geographers to assess before engaging in PRA with local people, including inadvertently exposing participants to danger or taking up their valuable time. Chambers mentions that facilitators must be aware that they raise expectations of a benefit, "even when the outsider goes to pains to explain that they have nothing to offer and nothing will follow from their visit." Another ethical concern is "Extracting information only for the outsiders’ benefit without **this being clear to those who provide it**," implying that it is acceptable to do as long as participants are aware.

As I read this, I wondered why outside geographers were entering Global South communities, potentially introducing risk and surely using time that locals could be using to do something else, for projects that will offer no benefit for the people involved.  If we use participatory and inclusive means for extractive ends, I question the value of such knowledge. 

Another aspect of Chamber's piece that could be improved upon was the alignment of the outsider-local dichotomy with the facilitator-participant dynamic. A facilitator need not be an outsider. While some mapping techniques would have to be introduced by outsiders, they could train locals to then teach others in the community. By empowering local community members, particularly those marginalized in the group, to pass on mapping knowledge, we can increase data accuracy and map quality (due to greater trust between facilitators and participants) as well as come closer to ensuring that projects are for the true benefit of community members. Ideally, PRA should be done by locals, for locals.

**Comment by nnhimichelle** [comment]
Thank you for this, it was interesting reading this! Ethics has always been put as my main concern whenever it comes to data collection, and I really enjoyed reading this and seeing your perspective. I agree with the approach for improved ethics, especially with greater trust comes with more valuable data that truly serves its purpose. There are people that say that it will be a tedious process, but it's a better approach for cartography and mapping over the long run!

**Comment by skarakas1** [comment]
Great comment! I think that Chambers was somewhat unclear on what the purposes of PM were in these communities. If it is simply to document and "know" more of the world, than that is not much different than the colonial projects of "exploration." On the other hand, I could definitely see situations where PM might be an important tool for making claims to space, for instance in Guatemala, where extractivist mining corporations displace indigenous people through land grabs. I think the author asks important questions, but more context in the article would have been helpful.
